server/Makefile

```diff
@@ -0,0 +1,13 @@
+-include .env
+
+.PHONY: clean
+clean: ## clean build artifacts
+	rm -r ${BUILD_DIR} 2> /dev/null || :
+
+.PHONY: build
+build: clean ## build release file
+	(ls build 2> /dev/null || mkdir ${BUILD_DIR}) &&\
+	rustup target add ${BUILD_TARGET} &&\
```

Comment:
what's the reason for using `&&` here? It will fail if we run cargo build in the background on a target that we're adding in the foreground. I think we should just run these sequentially

Reply:
`&&` means sequentially, but run next command only if previous completed successfully

Reply:
Yeah, it's`&` that detaches from a process